src/components/Home/CreateRoomButton.jsx

```diff
@@ -32,8 +33,10 @@ function CreateRoomButton() {
     });
 
     // Since we are the ones that made the room, make us the host in our redux.
+    let hostRef = ref(database, `rooms/${roomId}/host`);
```

Comment:
im not sure why its showing these changes in my branch. i didn't touch these files! maybe because they came from main when josh merged my host disconnect pr?